chapters/data-collection.tex

```diff
@@ -148,64 +105,52 @@ \subsection{Data licensing agreements}
+terms of use should be agreed on with the data owner, 
+typically through a data licensing agreement\sidenote{need to add a DIME wiki page}.
+Keep in mind that the data owner is likely not familiar
+with the research process, and therefore may be surprised
+at some of the things you want to do if you are not clear up front.
+You will typically want intellectual property rights to all research outputs developed used the data,
+a license for all uses of derivative works, including public distribution 
+(unless ethical considerations contraindicate this).
+This is important to allow the research team to store, catalog, and publish, in whole or in part,
+either the original licensed dataset or datasets derived from the original.
+Make sure that the license you obtain from the data owner allows these uses,
+and that you consult with the owner if you foresee exceptions with specific portions of the data.
 
-Another important consideration at this stage is 
-proper documentation and cataloging of data and associated metadata.
-It is not always clear what pieces of information jointly constitute a ``dataset'',
-and many of the sources you receive data from will not be organized for research.
-To help you keep organized and to put some structure on the materials you will be receiving,
-you should always retain the original data as received
-alongside a copy of the corresponding ownership agreement or license.
-You should make a simple ``readme'' document noting the date of receipt,
-the source and recipient of the data, and a brief description of what it is.
-All too often data produced by systems is provided as vaguely-named spreadsheets,
-or transferred as electronic communications with non-specific titles,
-and it is not possible to keep track of these kinds of information as data over time.
-Eventually, you will want to make sure that you are creating a collection or object
-that can be properly submitted to a data catalog and given a reference and citation.
-The metadata - documentation about the data - is critical for future use of the data.
-Metadata should include documentation of how the data was created, 
-what they measure, and how they are to be used.
-In the case of survey data, this includes the survey instrument and associated manuals; 
-the sampling protocols and field adherence to those protocols, and any sampling weights;
-what variable(s) uniquely identify the dataset(s), and how different datasets can be linked;
-and a description of field procedures and quality controls. 
-We use as a standard the Data Documentation Initiative (DDI), which is supported by the 
-World Bank's Microdata Catalog.\sidenote{\url{https://microdata.worldbank.org}}
+The World Bank has a template Data License Agreement which DIME follows.\sidenote{ \url{https://worldbankgroup.sharepoint.com/teams/ddh/SiteAssets/SitePages/ddh/DataLicenseAgreementTemplate_v4.pdf?cid=68a54269-bbff-4b47-846d-cab248ad7de1}}
```

Comment:
```suggestion
The World Bank has a template Data License Agreement which DIME follows.\sidenote{
\url{https://worldbankgroup.sharepoint.com/teams/ddh/SiteAssets/SitePages/ddh/DataLicenseAgreementTemplate_v4.pdf?cid=68a54269-bbff-4b47-846d-cab248ad7de1}}
```
just a line break so the line is not so wide. We cannot have line breaks before `\sidenote{` as it creates unwanted white space but right afterwards work.